front/lib/swr/spaces.ts

```diff
@@ -782,3 +783,82 @@ export function useSpacesSearch({
     nextPageCursor: data?.nextPageCursor || null,
   };
 }
+
+export function useSpaceSearchWithInfiniteScroll({
```

Comment:
Random question: Is there a reason you created a new hook and not extended the existing `useSpacesSearch` ?

Reply:
hm they are actually quite different and I think it's better have separate hooks for them? 🤔